tracker/common/dcp_agents/tests/test_analysis_agent.py

```diff
@@ -0,0 +1,73 @@
+import pytest
```

Comment:
@sampierson will be combining all the dcp agents in dcp, dcp-diag, and this repo into dcplib. He can move these tests when that is done

Reply:
That sound like a good move. So what do we want to do with these test cases now? Should we take them out or just leave them here and @sampierson could make a PR later to move them?

Reply:
I think its better to just merge this in and then Sam and copy/modify as needed